modules/nw-osp-configuring-external-load-balancer.adoc

```diff
@@ -32,77 +32,158 @@ on {rh-openstack-first}
 endif::[]
 to use an external load balancer in place of the default load balancer.
 
-You can also configure an {product-title} cluster to use an external load balancer that supports multiple subnets. If you use multiple subnets, you can explicitly list all the IP addresses in any networks that are used by your load balancer targets. This configuration can reduce maintenance overhead because you can create and destroy nodes within those networks without reconfiguring the load balancer targets.
+[IMPORTANT]
+====
+Configuring an external load balancer depends on your vendor's load balancer.
+
+The information and examples in this section are for guideline purposes only. Consult the vendor documentation for more specific information about the vendor's load balancer.
+====
+
+Red Hat supports the following services for an external load balancer:
+
+* OpenShift API
+* Ingress Controller
 
-If you deploy your ingress pods by using a machine set on a smaller network, such as a `/27` or `/28`, you can simplify your load balancer targets.
+You can choose to configure one or both of these services for an external load balancer, with configuring only the Ingress Controller service as a common configuration route.
 
-[NOTE]
+The following routes exist for configuring an external load balancer:
+
+* Use a node selector to map the Ingress Controller to a specific set of nodes, with  infrastructure nodes being a common node type that receives this configuration. You must then assign a static IP address to each node in this set, or configure each node to receive the same IP address from the Dynamic Host Configuration Protocol (DHCP).
+* Target all IP addresses on a subnet. This configuration can reduce maintenance overhead,because you can create and destroy nodes within those networks without reconfiguring the load balancer targets. If you deploy your ingress pods by using a machine set on a smaller network, such as a `/27` or `/28`, you can simplify your load balancer targets.
++
+[TIP]
 ====
-You do not need to specify API and Ingress static addresses for your installation program. If you choose this configuration, you must take additional actions to define network targets that accept an IP address from each referenced vSphere subnet.
+You can list all IP addresses that exist in a network by checking the machine config pool's resources.
 ====
 
-.Prerequisites
+.Considerations
 
-* On your load balancer, TCP over ports 6443, 443, and 80 must be reachable by all users of your system that are located outside the cluster.
+* For a front-end IP address, you could use the same IP address for the front-end IP address, the Ingress Controller's load balancer and API load balancer. Check the vendor's documentation for this capability.
```

Comment:
Per IBMSG, don't use "could":
Do not use in the present tense to mean might or can. Use only in the past conditional form, for example, “files that could not be identified”.

Replace with "can", or rewrite to something like:

"If your vendor supports X, you can..."